code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -71,7 +71,9 @@ client/proc/one_click_antag()
 
 		var/mob/living/carbon/human/candidate
 
-		for (var/i = 1 to min(candidates.len, 3))
+		var/antag_number = input("How many antags would you like to create?","Create Antagonists") as num|null
```

Comment:
Check if this is 0/null (`if (!antag_number)`) and return if it is.

The loop still wouldn't execute but it's nice to be explicit.

Reply:
Just `return 0` or an error message ?

Reply:
Error message would be nice too actually.